charts/iperf3/templates/service.yml

```diff
@@ -15,6 +15,6 @@ spec:
     app.kubernetes.io/instance: {{.Release.Name}}
   ports:
     - protocol: TCP
-      port: {{.Values.service.port}}
-      targetPort: 40000
+      port: {{.Values.service.port | default 5201}}
```

Comment:
please do not use default here and if, see above, for pods we would need a dedicated variable